### ISSUE TITLE
No email notification if Meeting Details or Title of invite are changed

### ISSUE THREAD

**Issue by sascha08-15** [nextcloud/calendar#4139]
### Steps to reproduce

1. Create an invite
2. Add external mail address to attend the meeting
3. Receive email confirmation on external address (working)
4. Change title or description
5. Bug: neither title nor description are updated for the attendee(s)

### Expected behavior

User should get a choice (like in outlook) to chose if updates should be send to attendee(s) of invite

### Actual behaviour

No mail notifications are sent, only if appointment is cancelled.

### Calendar app version

3.2.2

### Nextcloud version

latest stable

**Comment by tcitworld** [comment]
Related: https://github.com/nextcloud/calendar/issues/4019

**Comment by satyamkapoor** [comment]
> ### Steps to reproduce
> 
>     1. Create an invite
> 
>     2. Add external mail address to attend the meeting
> 
>     3. Receive email confirmation on external address (working)
> 
>     4. Change title or description
> 
>     5. Bug: neither title nor description are updated for the attendee(s)
> 
> 
> ### Expected behavior
> 
> User should get a choice (like in outlook) to chose if updates should be send to attendee(s) of invite
> ### Actual behaviour
> 
> No mail notifications are sent, only if appointment is cancelled.
> ### Calendar app version
> 
> 3.2.2

I have the same issue. Not sure why but in memory I remember this used to work before. With an email Invitation updated. But now, it send this invite updated email only in case I delete the event :/

**Comment by sascha08-15** [comment]
Well as a consultant I can just say any updates to the invite must be sent to all parties. 
One thing that could work is to have the updates sent out via mail conditionally. Outlook does something like that "notify all or just added party" (exact wording is different). 
What does not work is that location that includes relevant info is silently not shared but saved to local calendar. Rather unacceptable if you work with outside nextcloud users (e.g. Ms teams)

**Comment by Markoise** [comment]
This is one reason why I don't switch from Google Calendar to NC Calendar. 😞
Please could you send updates to all attendees? 🙏

**Comment by sascha08-15** [comment]
Any update on this? I recently sent an invite via NC - this becomes a good reason to go to MS Teams guys, IT is a no-go that someone does not get a notification. And even if it is not delivered, one must know asap (NC does silently not deliver). In the latest version, this is not fully resolved, really.

**Comment by QPanissod** [comment]
Agreed, but as a small company we probably do not have the means to help
with NC's development.

We are many making efforts to change our habits to use NC rather than the
mainstream proprietary solutions, but in order to also onboard our teams
and partners, we need to get organized to support solving these pain
points: changing our ways using the calendar and/or supporting dev effort
collectively.

What are the options that we can organize as small actors to help solving
those few pain points (external invites & shared calendar management ;
invite details ; invitate feedback)?

Quentin PANISSOD
+33 6 66 39 70 92


Le sam. 7 oct. 2023 à 22:32, sascha08-15 ***@***.***> a
écrit :

**Comment by SebastianKrupinski** [comment]
Fixed with https://github.com/nextcloud/server/pull/48583
